[2022-12-26T21:21:13.383+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: final_project_dag.extract_test manual__2022-12-26T21:21:06.233256+00:00 [queued]>
[2022-12-26T21:21:13.434+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: final_project_dag.extract_test manual__2022-12-26T21:21:06.233256+00:00 [queued]>
[2022-12-26T21:21:13.437+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-26T21:21:13.441+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2022-12-26T21:21:13.444+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-26T21:21:13.513+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract_test> on 2022-12-26 21:21:06.233256+00:00
[2022-12-26T21:21:13.529+0000] {standard_task_runner.py:55} INFO - Started process 809 to run task
[2022-12-26T21:21:13.535+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'final_project_dag', 'extract_test', 'manual__2022-12-26T21:21:06.233256+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/final_project_dag.py', '--cfg-path', '/tmp/tmprzk4mzsn']
[2022-12-26T21:21:13.541+0000] {standard_task_runner.py:83} INFO - Job 121: Subtask extract_test
[2022-12-26T21:21:18.695+0000] {task_command.py:389} INFO - Running <TaskInstance: final_project_dag.extract_test manual__2022-12-26T21:21:06.233256+00:00 [running]> on host 3dc0e72111ec
[2022-12-26T21:21:19.574+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kelompok 3
AIRFLOW_CTX_DAG_ID=final_project_dag
AIRFLOW_CTX_TASK_ID=extract_test
AIRFLOW_CTX_EXECUTION_DATE=2022-12-26T21:21:06.233256+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-26T21:21:06.233256+00:00
[2022-12-26T21:21:19.590+0000] {conn.py:372} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.25.0.9', 9092)]>: connecting to broker:9092 [('172.25.0.9', 9092) IPv4]
[2022-12-26T21:21:19.593+0000] {conn.py:1049} INFO - Probing node bootstrap-0 broker version
[2022-12-26T21:21:19.603+0000] {conn.py:401} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.25.0.9', 9092)]>: Connection complete.
[2022-12-26T21:21:19.728+0000] {conn.py:1106} INFO - Broker version identifed as 1.0.0
[2022-12-26T21:21:19.731+0000] {conn.py:1108} INFO - Set configuration api_version=(1, 0, 0) to skip auto check_version requests on startup
[2022-12-26T21:21:19.753+0000] {subscription_state.py:171} INFO - Updating subscribed topics to: ('projek_akhir',)
[2022-12-26T21:21:20.855+0000] {streaming.py:217} ERROR - Stream encountered HTTP error: 429
[2022-12-26T21:21:20.863+0000] {streaming.py:111} ERROR - HTTP error response text: {"title":"ConnectionException","detail":"This stream is currently at the maximum allowed connection limit.","connection_issue":"TooManyConnections","type":"https://api.twitter.com/2/problems/streaming-connection"}
[2022-12-26T21:22:21.332+0000] {streaming.py:217} ERROR - Stream encountered HTTP error: 429
[2022-12-26T21:22:21.336+0000] {streaming.py:111} ERROR - HTTP error response text: {"title":"ConnectionException","detail":"This stream is currently at the maximum allowed connection limit.","connection_issue":"TooManyConnections","type":"https://api.twitter.com/2/problems/streaming-connection"}
[2022-12-26T21:22:26.830+0000] {local_task_job.py:224} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2022-12-26T21:22:26.849+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 809. PIDs of all processes in the group: [809]
[2022-12-26T21:22:26.854+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 809
[2022-12-26T21:22:26.858+0000] {taskinstance.py:1483} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-12-26T21:22:26.870+0000] {streaming.py:203} ERROR - Stream encountered an exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/tweepy/streaming.py", line 120, in _connect
    sleep(http_error_wait)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1485, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2022-12-26T21:22:26.918+0000] {logging_mixin.py:137} INFO - Disconnected
[2022-12-26T21:22:26.931+0000] {streaming.py:193} INFO - Stream disconnected
[2022-12-26T21:22:26.994+0000] {conn.py:372} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.25.0.9', 9092)]>: connecting to broker:9092 [('172.25.0.9', 9092) IPv4]
[2022-12-26T21:22:27.110+0000] {conn.py:401} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:9092 <connecting> [IPv4 ('172.25.0.9', 9092)]>: Connection complete.
[2022-12-26T21:22:27.255+0000] {cluster.py:376} INFO - Group coordinator for consumers is (1, 'localhost', 9092, None)
[2022-12-26T21:22:27.257+0000] {base.py:688} INFO - Discovered coordinator 1 for group consumers
[2022-12-26T21:22:27.259+0000] {base.py:735} INFO - Starting new heartbeat thread
[2022-12-26T21:22:27.263+0000] {consumer.py:342} INFO - Revoking previously assigned partitions () for group consumers
[2022-12-26T21:22:27.269+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:27.372+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:27.374+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:27.376+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:27.479+0000] {base.py:710} WARNING - Marking the coordinator dead (node 1) for group consumers: Node Disconnected.
[2022-12-26T21:22:27.480+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:27.483+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:27.484+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:27.486+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:27.590+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:27.694+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:27.696+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:27.698+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:27.801+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:27.805+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:27.812+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:27.817+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:27.926+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:28.032+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:28.036+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:28.041+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:28.147+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:28.151+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:28.152+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:28.155+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:28.413+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:28.417+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:28.420+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:28.426+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:28.686+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:28.701+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:28.705+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:28.708+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:29.133+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:29.142+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:29.149+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:29.159+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:29.536+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:29.545+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:29.551+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:29.555+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:30.407+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:30.434+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:30.438+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:30.446+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:31.284+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:31.300+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:31.307+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:31.313+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:32.342+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:32.360+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:32.364+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:32.365+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:33.371+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:33.381+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:33.388+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:33.391+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:34.533+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:34.555+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:34.563+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:34.581+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:35.672+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:35.679+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:35.689+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:35.724+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:36.709+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:36.730+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:36.733+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:36.738+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:37.660+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:37.665+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:37.673+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:37.679+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:38.771+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:38.856+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:38.871+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:38.913+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:39.830+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:39.847+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:39.853+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:39.870+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:41.022+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:41.029+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:41.034+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:41.037+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:42.212+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:42.218+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:42.221+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:42.260+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:43.165+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:43.183+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:43.189+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:43.194+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:44.094+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:44.124+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:44.129+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:44.135+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:45.081+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:45.113+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:45.118+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:45.121+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:46.035+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:46.038+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:46.041+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:46.049+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:47.194+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:47.204+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:47.209+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:47.211+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:48.335+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:48.341+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:48.345+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:48.348+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:49.382+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:49.409+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:49.412+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:49.420+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:50.401+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:50.410+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:50.412+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:50.413+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:51.389+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:51.394+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:51.397+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:51.403+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:52.372+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:52.377+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:52.379+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:52.381+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:53.406+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:53.409+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:53.411+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:53.416+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:54.440+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:54.444+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:54.446+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:54.450+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:55.524+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:55.527+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:55.529+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:55.531+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:56.601+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:56.603+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:56.604+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:56.605+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:57.523+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:57.528+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:57.530+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:57.532+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:58.449+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:22:58.451+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:22:58.452+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:22:58.454+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:22:59.481+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:22:59.484+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:22:59.487+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:22:59.488+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:00.506+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:00.507+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:00.508+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:00.510+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:01.438+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:01.440+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:01.441+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:01.442+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:02.359+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:02.361+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:02.363+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:02.365+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:03.591+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:03.593+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:03.594+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:03.595+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:04.816+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:04.826+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:04.829+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:04.833+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:06.017+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:06.026+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:06.031+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:06.038+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:07.221+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:07.225+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:07.227+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:07.229+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:08.161+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:08.187+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:08.191+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:08.215+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:09.093+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:09.096+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:09.098+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:09.100+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:10.180+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:10.189+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:10.196+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:10.211+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:11.303+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:11.304+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:11.305+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:11.306+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:12.394+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:12.404+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:12.410+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:12.415+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:13.460+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:13.467+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:13.470+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:13.473+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:14.446+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:14.461+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:14.467+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:14.477+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:15.444+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:15.453+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:15.456+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:15.459+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:16.596+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:16.604+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:16.607+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:16.611+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:17.730+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:17.750+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:17.759+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:17.772+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:18.681+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:18.685+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:18.689+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:18.695+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:19.657+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:19.743+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:19.748+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:19.755+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:20.759+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:20.773+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:20.775+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:20.780+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:21.868+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:21.874+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:21.878+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:21.882+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:23.078+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:23.084+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:23.091+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:23.094+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:24.252+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:24.263+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:24.265+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:24.271+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:25.462+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2022-12-26T21:23:25.468+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 111. Disconnecting.
[2022-12-26T21:23:25.473+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2022-12-26T21:23:25.477+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:26.653+0000] {conn.py:372} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2022-12-26T21:23:26.666+0000] {conn.py:410} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 99. Disconnecting.
[2022-12-26T21:23:26.675+0000] {conn.py:811} INFO - <BrokerConnection node_id=1 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2022-12-26T21:23:26.678+0000] {client_async.py:326} WARNING - Node 1 connection failed -- refreshing metadata
[2022-12-26T21:23:26.860+0000] {process_utils.py:147} WARNING - process psutil.Process(pid=809, name='airflow task ru', status='sleeping', started='21:21:12') did not respond to SIGTERM. Trying SIGKILL
[2022-12-26T21:23:26.957+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGKILL to group 809
[2022-12-26T21:23:27.420+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=809, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='21:21:12') (809) terminated with exit code Negsignal.SIGKILL
[2022-12-26T21:23:27.429+0000] {standard_task_runner.py:170} ERROR - Job 121 was killed before it finished (likely due to running out of memory)
